rpc/documentation/api.md

```diff
@@ -1393,17 +1394,67 @@ transaction using a wallet private keys.
 
 **Expected errors:**
 
-- `InvalidArgument`: The serialized transaction can not be decoded.
-
 - `Aborted`: The wallet database is closed.
 
 - `InvalidArgument`: The private passphrase is incorrect.
 
+- `InvalidArgument`: The serialized transaction can not be decoded.
+
 **Stability:** Unstable: It is unclear if the request should include an account,
   and only secrets of that account are used when creating input scripts.  It's
   also missing options similar to Core's signrawtransaction, such as the sighash
   flags and additional keys.
 
+  ___
+
+  #### `SignTransactions`
+
+  The `SignTransactions` method adds transaction input signatures to a set of
+  serialized transactions using a wallet private keys.
+
+  **Request:** `SignTransactionsRequest`
+
+  - `bytes passphrase`: The wallet's private passphrase.
+
+  - `repeated UnsignedTransaction unsigned_transaction`: - The unsigned transactions set.
+
+    **Nested message:** `UnsignedTransaction`
+
+    - `bytes serialized_transaction`: The transaction to add input signatures to.
+
+    - `repeated uint32 input_indexes`: The input indexes that signature scripts must
+    be created for.  If there are no indexes, input scripts are created for every
+    input that is missing an input script.
+
+    - `repeated AdditionalScript additional_scripts`: Additional output scripts of
```

Comment:
this is indented too far, it should be at the same indent level as `repeated UnsignedTransaction unsigned_transaction`.